wvd-templates/Create and provision WVD host pool/mainTemplate.json

```diff
@@ -292,81 +357,170 @@
                 "platformFaultDomainCount": 2
             },
             "sku": {
-                "name": "[variables('avSetSKU')]"
+                "name": "Aligned"
```

Comment:
I think this needs to be "Classic" if rdshManagedDisks is true, see avSetSKU in the previous version of this file.

Reply:
you mean it should be "Classic" if rdshManagedDisks is **false** ?

Reply:
Yes, that is what I meant.